Client/Auth/index.ts

```diff
@@ -101,6 +101,32 @@ export class Auth {
 		}
 		return result
 	}
+
+	isAssumed(): boolean {
+		const data = this.data()
+		return data.user?.organisation?.code != data.organisation
+	}
+
+	async assume(code: string) {
+		const result = await this.connection.get<model.LoginResponse, 400 | 403 | 404 | 500>(`auth/assume/org/${code}`)
+		if (!isError(result)) {
+			this.connection.token = result.token
+			window.sessionStorage.setItem("authData", JSON.stringify(result))
+		}
+		return result
+	}
+
+	async unassume() {
+		const data = this.data()
+		let result: model.LoginResponse | (model.ErrorResponse & { status: 400 | 403 | 404 | 500 | 503 }) | undefined
+		if (data.user?.organisation?.code)
+			result = await this.assume(data.user.organisation.code)
+		else
+			result = undefined
```

Comment:
the if isnt necessary, that will always be present

Reply:
not sure when it would be happening but in case there would be an empty object returning from this.data() i wanted to be sure